cincinnati/src/common.rs

```diff
@@ -0,0 +1,8 @@
+/// Adds "/" if the input is not empty
+pub fn parse_path_namespace(path_namespace: &str) -> String {
+    if path_namespace == "" {
+        path_namespace.to_string()
+    } else {
+        "/".to_owned() + path_namespace
```

Comment:
You may want to take care of existing leading and trailing slashes, to avoid having (leading or trailing) `//` in the final string.

Reply:
The docstring mentions that it must be provided without leading `/`, but I can handle it in the code as well if you think it's necessary.

Reply:
Not necessary (can be done at any later point if we still want), but if this become deployable by the general audience, I can foresee human mistakes (or plainly not reading the help docs). Feel free to leave it for now.